### PR TITLE
bfs: add page

### PR DESCRIPTION
**Pull request by MANICX100**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/bfs.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Breadth-first search for your files.
+> More information: <https://manned.org/bfs>.
+
+- Find files by extension:
+
+`bfs {{root_path}} -name '{{*.ext}}'`
+
+- Find files matching multiple path/name patterns:
+
+`bfs {{root_path}} -path '{{**/path/**/*.ext}}' -or -name '{{*pattern*}}'`
```
Comment (spageektti) [inline pages/common/bfs.md:12]:
```suggestion
`bfs {{root_path}} -path '{{**/path/**/*.ext}}' -or -name '{{pattern}}'`
```
What's the point of the * symbols in "pattern"?
if they were meant to make the text **bold** or _italics_, it is not supported.

Comment (MANICX100) [reply pages/common/bfs.md:12]: It means substring as in anywhere in the text of name has pattern *wildcard from start - *wildcard to end etc

Comment (spageektti) [reply pages/common/bfs.md:12]: Thanks for explaining.

**Review by spageektti — APPROVED** [review]
Thanks for your contribution. Welcome to tldr-pages!

**Pull request merged**